### ISSUE TITLE
Doesn't support env var --> SYNTHTOOL_GOOGLEAPIS (local googleapis)

### ISSUE THREAD

**Issue by munkhuushmgl** [googleapis/google-cloud-dotnet#3339]
**Is your feature request related to a problem? Please describe.**
It doesn't support $SYNTHTOOL_GOOGLEAPIS (env var that points to local googleapis/googleapis).  Other languages supports $SYNTHTOOL_GOOGLEAPIS, and I thnk it might be good idea to add this as option and keep it consistent throughout all languages.

 **Describe the solution you'd like**
Add an option to read $SYNTHTOOL_GOOGLEAPIS (points to local googleapis) if that env variable exists.

**Comment by jskeet** [comment]
Most of our generation tooling predates synthtool, and assumes `googleapis` is directly under the root directory - cloning it if necessary. Changing that might be significant work - mostly in terms of tracking down everywhere I *expect* to find googleapis there. Is there a concrete use case that would make that work worthwhile?

**Comment by munkhuushmgl** [comment]
The use case is to be able to edit .proto files locally and generate libraries from them in more than 1 language. I generate libraries in multiple languages _from local .proto files_ with changes, so there should be 1 common location on my computer where googleapis lives.

**Comment by jskeet** [comment]
Okay, I'll take a look. No promises just yet though.

**Comment by jskeet** [comment]
Okay, I've created #3340 and tested it with a copy of googleapis with a change to KMS.
It's likely this will be the only synthtool environment variable we'll implement for now:

- SYNTHTOOL_ARTMAN_VERSION: We don't use Artman at all
- SYNTHTOOL_GENERATOR: It's likely to be Chris (or myself) working on generator changes anyway. Once we've moved over to the microgenerator wholeheartedly, we might revisit this if it becomes useful.
